### ISSUE TITLE
Print Preview not rendering images

### ISSUE THREAD

**Issue by rabgc** [ipython/ipython#8648]
This issue was raised before, but it was closed without resolving the issue, at least for me.  I have stored a local copy of IPy_header.png next to my notebook.  Referring to http://ipython.org/_static/IPy_header.png works as expected, rendering images in both the notebook and Print Preview, but a reference to IPy_header.png does not render the image in Print Preview.

**Comment by lambdalisue** [comment]
+1

When I access the URL. It say

```
400 : Bad Request

The error was:

Not a notebook: a_correct_path_of/image.png
```

**Comment by takluyver** [comment]
Should be fixed in 4.0 - see jupyter/notebook#228
